spec/lib/api/2.0/skus-spec.js

```diff
@@ -77,11 +77,13 @@ describe('Http.Api.Skus.2.0', function() {
                 }
             }
         ],
+        ibms: [],
         autoDiscover: "false",
         identifiers: [],
         createdAt: '010101',
         updatedAt: '010101',
         tags: [],
```

Comment:
I meant that since it's supposed to be a link, should the test object be a link here, to match [nodes-spec](https://github.com/brianparry/on-http/blob/70e187dd2ede3cbefc1620d87712188b5e0d924d/spec/lib/api/2.0/nodes-spec.js#L107)?